app/javascript/src/components/Dashboard/Contacts/constants.js

```diff
@@ -21,15 +21,15 @@ export const CONTACTS_FORM_VALIDATION_SCHEMA = yup.object().shape({
     .required("Email is required"),
   role: yup
     .object()
-    .nullable()
-    .required("Role is required")
     .shape({
       label: yup.string().oneOf(ROLES.map(role => role.label)),
       value: yup.string().oneOf(ROLES.map(role => role.value)),
-    }),
+    })
+    .nullable()
+    .required("Role is required"),
 });
 
-export const contactListData = setShowDeleteAlert => [
+export const CONTACT_LIST_DATA = setShowDeleteAlert => [
```

Comment:
If this is required to be a function, then you can't name it as a literal constant. Name it as you ought to name a function.